src/object_properties.cpp

```diff
@@ -159,6 +162,7 @@ void ObjectProperties::deSerialize(std::istream &is)
 			nametag_color = readARGB8(is);
 			automatic_face_movement_max_rotation_per_sec = readF1000(is);
 			infotext = deSerializeString(is);
+			no_static_save = readU8(is);
```

Comment:
hm, however the client does not need to know about this so it should probably be removed.